src/nomad_material_processing/solution/general.py

```diff
@@ -937,3 +937,6 @@ def normalize(self, archive: 'EntryArchive', logger: 'BoundLogger') -> None:
         if not self.solution:
             self.solution = SolutionReference()
         self.solution.reference = self.create_solution_entry(solution, archive, logger)
+
+
+m_package.__init_metainfo__()
```

Comment:
This is the reason why in some circumstances the `materials_processing_plugin` alias worked and the `materials_processing_plugin.solution` alias did not.

Reply:
Indeed, this resolves the issue with aliasing on my local.